test/integration/sap_hana_scaleout/sap_hana_scaleout_test.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sap_hana_scaleout
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/gcloud"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/tft"
+	"github.com/stretchr/testify/assert"
+	"github.com/terraform-google-modules/terraform-google-sap/test/integration/common"
+)
+
+func TestSapHanaScaleoutModule(t *testing.T) {
+	sapHanaSc := tft.NewTFBlueprintTest(t)
+
+	sapHanaSc.DefineVerify(func(assert *assert.Assertions) {
+		sapHanaSc.DefaultVerify(assert)
+
+		instanceSelfLinks := make(map[string]string)
+		instanceSelfLinks["primary"] = sapHanaSc.GetStringOutput("sap_hana_primary_self_link")
+
+		workersListRaw := sapHanaSc.GetStringOutput("hana_scaleout_worker_self_links")
+		workers := strings.Split(strings.Trim(workersListRaw, "[]"), ",")
+		for i, workerLink := range workers {
+			instanceSelfLinks[fmt.Sprintf("worker-%d", i)] = workerLink
+		}
```

Comment:
We should directly get this as a slice here and throughout. Example:
https://github.com/terraform-google-modules/terraform-google-address/blob/a4d0830ce1b4b0f6a082f478fdbab1ccfed6615b/test/integration/dns_forward_example/dns_forward_example_test.go#L34